google/cloud/storage/internal/curl_client_test.cc

```diff
@@ -24,6 +24,10 @@
 #include <utility>
 #include <vector>
 
+#ifndef INSTANTIATE_TEST_SUITE_P
+#error "Wrong googletest TEST_SUITE_P not defined"
```

Comment:
INSTANTIATE_TEST_SUITE_P?

Reply:
Actually this is debugging code, removed.